src/coreclr/debug/daccess/daccess.cpp

```diff
@@ -7502,12 +7496,8 @@ BOOL OutOfProcessExceptionEventGetProcessIdAndThreadId(HANDLE hProcess, HANDLE h
     // UNIXTODO: mikem 1/13/15 Need appropriate PAL functions for getting ids
     *pPId = (DWORD)(SIZE_T)hProcess;
     *pThreadId = (DWORD)(SIZE_T)hThread;
-#else
-#if !defined(FEATURE_CORESYSTEM)
-    HMODULE hKernel32 = WszGetModuleHandle(W("kernel32.dll"));
 #else
 	HMODULE hKernel32 = WszGetModuleHandle(W("api-ms-win-core-processthreads-l1-1-1.dll"));
```

Comment:
Both `GetProcessIdOfThread` and `GetThreadId` methods are available on Windows Vista. We can just reference them statically. No need for dynamic loading.